apps/src/applab/api.js

```diff
@@ -328,8 +328,8 @@ export function startWebRequest(url, func) {
   return Applab.executeCmd(null, 'startWebRequest', {url: url, func: func});
 }
 
-export function startWebRequestSync(url, headers, func) {
```

Comment:
it looks like this param was added in #26383 to allow levelbuilders/users to make requests that require headers. i'm not sure if this is still needed, but is there a way for us to double-check? (i honestly don't know the answer to that, but we may be able to ask curriculum?)